packages/itwin/property-grid/package.json

```diff
@@ -38,13 +38,14 @@
     "rebuild": "npm run clean && npm run build"
   },
   "dependencies": {
+    "@itwin/core-i18n": "rc",
```

Comment:
should these be peerDeps?

Reply:
after writing that, makes me question, if we could just get away with making it a dev dep since its only used for types

Reply:
i was thinking that as well....maybe test that? if it builds, should be fine

Reply:
this seemed to work. any pkgs were just using as types i kept as a dev dep & removed as a peer dep.
also went through and removed any peers of peers as that should be unnecessary